site/docs/src/json/identity-providers/links/create-link-request-with-identityProviderLink-in-body.json

```diff
@@ -0,0 +1,8 @@
+{
+  "identityProviderLink":{
```

Comment:
I might rename the corresponding other file for this request with a `deprecated` prefix just to make it clearer.